### DISCUSSION TITLE
Update from C2531 to CC2652R

### DISCUSSION THREAD

**Discussion by tb205gti** [Koenkk/zigbee2mqtt#10354] [Devices]
Old coordinator: C2531, running 20211115
New coodinator: CC2652R zig-zig-ah!, running 20210708
Zigbee2mqtt version: 1.22.1, running on Ubuntu

I have flashed the zig-a-zig-ah! with the latest coordinator fw, and i have flashed the same ieee address as the old coordinator. Changed the config to reflect the new coordinator. Zigbee2mqtt starts fine - but there are no connections in the network. The network map just shows all my devices, but none are connected. The log has errors as "no route to host".

Now; I thought that I could upgrade the coordinator from C2531 to C2652R without loosing the network? 
Do I just need to give it some time to recover, or is there something else wrong? The C2531 has become rather unstable, so I'd like to get the new coordinator up and running again. I's rather not have to pair everything again..

**Comment by tb205gti** [comment]
So I have verified that the same ieee address is indeed in use, and I have verified that the same pan_id is being used. From the log

Old Coordinator:
{"commit":"c2b5229f","coordinator":{"meta":{"maintrel":3,"majorrel":2,"minorrel":6,"product":0,"revision":20211115,"transportrev":2},"type":"zStack12"},"log_level":"info","network":{"channel":11,"extendedPanID":"0x00124b0018ed3f7a","
panID":6754},"permit_join":true,"version":"1.22.1"}' 


New Coordinator:
{"commit":"c2b5229f","coordinator":{"meta":{"maintrel":1,"majorrel":2,"minorrel":7,"product":1,"revision":20210708,"transportrev":2},"type":"zStack3x0"},"log_level":"info","network":{"channel":11,"extendedPanID":"0x00124b0018ed3f7a",
"panID":6754},"permit_join":true,"version":"1.22.1"}'

**Comment by Koenkk** [comment]
You can find a guide here: https://www.zigbee2mqtt.io/guide/faq/#how-do-i-migrate-from-one-adapter-to-another

**Comment by tb205gti** [comment]
Yes, that is the guide I'm following :/

**Reply by Koenkk** [reply]
Try disconnecting some routers from the mains and reconnect, this should trigger the network rebuilding.

If that doesn't help provide the debug log when trying to control a switch or a bulb.
See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable debug logging.

**Comment by tb205gti** [comment]
That helped - I kept the network down for an hour - then it started working once I started it again.

However, a new issue. I'm using Ikea switches (E1743, sw: 2.2.010) quite a few places and almost everything is working very good. Apart from - and this has started after the update of the coordinator - brightness_move_down never gets reported. Once I let go of the button, I get a correct brightness_stop. Holding the brightness up gives me a correct brightness_move_up, and letting go a brightness_stop. This happens on all my Ikea switches now.

Also - again Ikea, but this time E1524/E1810 (1.2.214 & 2.3.014). arrow_right_click and arrow_left_click gets sent perfectly. But brightness_down_click and brightness_up_click only gets sent once I click arrow_left or arrow_right. It is not delayed - it just doesn't get sent at all. So kinda the same issue as brightness_move above.

Any hints/tips?

**Comment by tb205gti** [comment]
..btw I just verified that this is working with the old coordinator (C2531). so must be an issue with the CC2562.

**Comment by tb205gti** [comment]
CC2562 - no brightness_move_down command -> https://youtu.be/-VUyGF44Y_k
C2531 - working perfectly -> https://youtu.be/1xnjy3pJJEs

Everything is the same - only thing changed is the coordinator.

**Comment by Koenkk** [comment]
Try to update to the latest dev branch, force remove and repair the device after that, some issues regarding this have been fixed there (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by tb205gti** [comment]
Updated the coordinator to latest dev - updated the z2m to latest dev, still same issue :/

I'll try to remove one switch and re-pair it again, so see if that does anything.

**Comment by tb205gti** [comment]
OK, not able to pair the E1743 using the CC2562 coordinator - it never gets the pairing request. just sits until it times out. Switching back to the old coordinator i can pair again.

So there are two issues:


- CC2562 not able to pair E1743 switches at all
- brightness_move_down not handled properly on E1743
- (Issues with E1810, somewhat the same as for the E1743)

How can I help ? Debug info? Dev builds? anything?
